src/components/data_comps/datm/cime_config/config_component.xml

```diff
@@ -60,7 +60,7 @@
 
   <entry id="DATM_PRESAERO">
     <type>char</type>
-    <valid_values>none,clim_1850,clim_2000,clim_2010,trans_1850-2000,SSP1-1.9,SSP1-2.6,SSP2-4.5,SSP3-7.0,SSP4-3.4,SSP4-6.0,SSP5-3.4,SSP5-8.5,cplhist</valid_values>
+    <valid_values>none,clim_1850,clim_2000,clim_2010,trans_1850-2000,SSP1-1.9,SSP1-2.6,SSP2-4.5,SSP3-7.0,SSP4-3.4,SSP4-6.0,SSP5-3.4,SSP5-8.5,cesm2_omip,cplhist</valid_values>
```

Comment:
@fischer-ncar should these changes go to master?

Reply:
I'll let @klindsay28 answer about including CMIP6 OMIP compsets.

Reply:
I think that it's okay - I realized as I looked into it further - I withdraw the question.